### PR TITLE
✨ introducing `inputs_required` to workbench

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- ✨ introducing `inputs_required` to workbench

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
thanks

**Review by GitHK — APPROVED** [review]
Thanks for this!

File: packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py [hunk 1]
```diff
@@ -27,11 +27,12 @@ class NodeCreate(InputSchemaWithoutCamelCase):
 
 class NodePatch(InputSchemaWithoutCamelCase):
     service_version: ServiceVersion = FieldNotRequired(alias="version")
-    label: str = Field(None)
-    inputs: InputsDict = Field(None)
+    label: str = FieldNotRequired()
+    inputs: InputsDict = FieldNotRequired()
+    inputs_required: list[InputID] = FieldNotRequired(alias="inputsRequired")
```
Comment (GitHK) [inline packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py:32]: this reads better
```suggestion
    required_inputs: list[InputID] = FieldNotRequired(alias="inputsRequired")
```

File: packages/models-library/src/models_library/projects_nodes.py [hunk 2]
```diff
@@ -156,6 +156,11 @@ class Node(BaseModel):
     inputs: InputsDict | None = Field(
         default_factory=dict, description="values of input properties"
     )
+    inputs_required: list[InputID] = Field(
+        default_factory=list,
+        description="Input IDs required",
```
Comment (sanderegg) [inline packages/models-library/src/models_library/projects_nodes.py:161]:
```suggestion
        description="Defines inputs that are required in order to run the service",
```

**Review by sanderegg — APPROVED** [review]
thanks

File: packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py (hunk 1 above)
Comment (matusdrobuliak66) [reply packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py:32]: I would like to keep it consistent with other `inputs_*` naming we use

**Pull request merged**